BrownField/Addons/SRM/readme.md

```diff
@@ -32,6 +32,14 @@ cd ARM
 az deployment group create -g AVS-Step-By-Step-RG -n AVS-SRM-Deployment -c -f "SRM.deploy.json" -p "@SRM.parameters.json"
 ```
 
+### Azure PowerShell
+
+```powershell
+cd PowerShell
+
+.\deploy.ps1
```

Comment:
Would be good to mention that you'll need to edit the ps1 file to specify resource group and private cloud name, but besides that looks great!